index.d.ts

```diff
@@ -45,6 +45,18 @@ export interface ParseOptions {
 		//=> {foo: ['1', '2', '3']}
 		```
 
+	- `bracket-separator`: Parse arrays (that are explicitly marked with brackets) with elements separated by a custom character:
+
+		```
+		import queryString = require('query-string');
+
+		queryString.parse('foo[]=1', {arrayFormat: 'bracket-separator', arrayFormatSeparator: '|'});
+	 	//=> {foo: ['1']}
+
+		queryString.parse('foo[]=1|2|3', {arrayFormat: 'separator', arrayFormatSeparator: '|'});
```

Comment:
It would be more valuable to show 
```suggestion
		queryString.parse('foo[]=1', {arrayFormat: 'separator', arrayFormatSeparator: '|'});
```

So the user could also see the difference.

Reply:
I don't quite follow what this suggestion is recommended. Doesn't this suggestion modify the documentation to be the same as the other `bracket-separator` example?

As a note, I have just noticed that this second example incorrectly shows `arrayFormat: 'separator'` when it should be `arrayFormat: 'bracket-separator'`

Reply:
I'm not quite following what this is wanting.

 I did realize I accidently left the `arrayFormat` set to `separator` here instead of `bracket-separator`, so I've updated that as needed.